vispy/gloo/preprocessor.py

```diff
@@ -45,7 +45,7 @@ def replace(match):
                 logger.critical('"%s" not found' % filename)
                 raise RuntimeError("File not found", filename)
             text = '\n// --- start of "%s" ---\n' % filename
-            text += remove_comments(open(path).read())
+            text += open(path).read()
```

Comment:
The descriptors should be freed along with the File objects. Is that not the case?